tensorboard/webapp/metrics/views/card_renderer/scalar_card_test.ts

```diff
@@ -2376,6 +2376,24 @@ describe('scalar card', () => {
           }),
         ]);
       }));
+
+      it('does not render fobs when no timeSelection is provided', fakeAsync(() => {
+        store.overrideSelector(getMetricsLinkedTimeSelection, {
+          start: {step: 20},
+          end: null,
+        });
+        const fixture = createComponent('card1');
+        const scalarCardFobController = fixture.debugElement.query(
+          By.directive(ScalarCardFobController)
+        ).componentInstance;
+        delete scalarCardFobController.timeSelection;
+        fixture.detectChanges();
+        const fobController = fixture.debugElement.query(
+          By.directive(CardFobComponent)
+        ).componentInstance;
```

Comment:
nit: could you have new lines between 2393-2394? Have tests separated into blocks (set up, action, expect) helps with the reading.
I honestly am not sure where is the line between setup and action. line 2390-2391?

Reply:
Sure

Reply:
Would a line between line 2390-2391 make sense?